blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,81 @@
+
+These fears are common both in junior and in senior developers, but the truth is that we are all learning all the time, and the more we interact with others, the more we learn. I am not an exception, but I decided to open myself more to collaborative work for three main reasons:
+
+- I was not happy with the amount and quality of collaborative work in my teams,
+- I consider these beliefs mostly as prejudices, and
+- Every time I was approaching work in a more collaborative way, even though feeling unnatural, it showed to be quite powerful.
+
+Don't get me wrong, I am not talking about overdoing it. It is important to keep the balance, working many hours per day in groups is much more tiring, challenging and also more inefficient than just working on your own. So what we really want is to find the range where collaborative work produces a positive impact in our working life. In order for that to happen, it is crucial to accumulate experiences and get a feeling of which collaboration paradigm is better to use for each use case. Daily meeting? Knowledge sharing session? Second opinion? Think of a toolbox for team collaboration, this is exactly what is offered to you. Every person, every situation, every topic is different, that's what makes this journey exciting and why you want to have a variety of options and adapt each time.
+
+There are many opportunities to collaborate with a colleague while working on a task, and it doesn't need to be monotone or boring, or useless. It can be fun, productive, fulfilling, and yes, challenging if you are not used to do it, but meaningful and definitely worthy in any case. Nevertheless we keep working way too much in silos. Most of the time we use daily meetings or other team meetings, together with written communication (issue tracker system, pull Requests, chat tool) to grab feedback on our work or to discuss topics. They do not fit well all purposes though. 
+
+## In practice 
+
+As you will realise in this section, the common factors of any kind of collaborative work are: clear goals, preparation, focus, constantly challenging yourself and each other’s thoughts and a constructive attitude. 
+
+There are many situations that greatly benefit from doing tasks in small groups, here I am going to describe the ones that more resonate with me and that I imagine to be more valuable for you. Don't limit yourself to this list, but use it also as inspiration to find your own ways.
```

Comment:
pick me up. The headline tells me that it's going to be about how to start, but the text only talks about in a meta way about the following content without taking my hand and telling me what's up next

Reply:
After reading the next sections I would have hoped that you tell me "hey I will present you four amazing ways to collaborate with your team" and such.


The following headlines with emojis feel and look great but it's difficult to know what in them. For example "The doubtful solution " is actually about "the second opinion". I think you should be more clear about the content instead of creating creative headline to make it more accessible for people?

Reply:
For the first part I agree, it is not introducing you to what it is coming. I added a sentence similar to the one you mentioned and removed some bla bla.

Reply:
For the second part, I would prefer to not do any change on the headlines. For the first 2, by reading the first sentence the reader already know what is it about. I like the creative part of it, I don't consider it misleading, and I think it triggers the curiosity of the reader. For the latter 2, I think the title is quite self-explanatory.